wps-common/src/main/java/au/org/aodn/aws/util/EmailService.java

```diff
@@ -113,4 +122,76 @@ public static String getCompletedJobEmailTemplate() {
     public static String getFailedJobEmailTemplate() {
         return String.format("%s/%s", WpsConfig.EMAIL_TEMPLATES_LOCATION, WpsConfig.FAILED_JOB_EMAIL_TEMPLATE_NAME);
     }
+
+    public static String formatRequestDetail(SubsetParameters subsetParameters, String collection) {
+
```

Comment:
Would be nice to use a consistent style in java  - we don't normally put blank lines after {  - and its not done elsewhere in this PR.